desktop.ini

```diff
@@ -0,0 +1,4 @@
+[ViewState]
+Mode=
+Vid=
+FolderType=StorageProviderGeneric
```

Comment:
?

Reply:
I had changed the folder icon by mistake ,that's why it is generated.